ci_scripts/templates/.copyright.tmpl

```diff
@@ -0,0 +1,2 @@
+Copyright (c) 2020-${years} Arm Limited and Contributors. All rights reserved.
```

Comment:
It could be that we add a new file in 2021, does it need to follow this `2020-${years}

Reply:
With Apache-2.0 I'm pretty sure you have to retain the original copyright notice of the project. So 2020-2021 for a new file added in 2021 is fine.